src/client/app.html.mustache

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <base href="/">
```

Comment:
Possibly worth adding a comment in here that the `<base>` tag is needed to support Angular's `$location` service in HTML 5 mode. That will serve as a reminder that we should be able to drop this requirement in future.